Guide.docc/MigrationGuide.md

```diff
@@ -14,27 +14,27 @@
 
 Swift's concurrency system, introduced in [Swift 5.5](https://www.swift.org/blog/swift-5.5-released/),
 makes asynchronous and parallel code easier to write and understand.
-With the Swift 6 language mode, the compiler can now also
-guarantee that concurrent programs are data-race-free.
-When this mode is enabled, the compiler safety checks that were
-optional become required.
+With the Swift 6 language mode, the compiler can now 
+guarantee that concurrent programs are free of data races.
+With this mode enabled, compiler safety checks that were
+previously optional become required.
 
-Adopting the Swift 6 language mode is entirely under your control,
+Adopting the Swift 6 language mode is entirely under your control
 on a per-target basis.
-Targets that build with previous modes, as well as code that uses other
+Targets that build with previous modes, as well as code in other
 languages exposed to Swift, can all interoperate with
 modules that have been migrated to Swift 6.
 
-It is possible you have been incrementally adopting all concurrency features
+It is possible you have been incrementally adopting concurrency features
 as they were introduced.
 Or, you may have been waiting for Swift 6 to begin using them.
-Regardless of where your project is in this process, this guide exists to
-provide concepts and practical help to ease the migration.
+Regardless of where your project is in this process, this guide provides
+concepts and practical help to ease the migration.
 
 Here you will find articles and code examples that will:
 
 - Explain the concepts used by Swift's data-race safety model.
-- Help you enable complete concurrency checking with Swift 5 projects.
+- Show how to enable complete concurrency checking for Swift 5 projects.
```

Comment:
Previously, only one bullet point referenced 'you', this change makes this item have the same style as the rest.